src/main/kotlin/it/pagopa/wallet/domain/wallets/details/LastFourDigits.kt

```diff
@@ -0,0 +1,14 @@
+package it.pagopa.wallet.domain.wallets.details
+
+data class LastFourDigits(val lastFourDigits: String) {
```

Comment:
Not related to this PR, but have you ever considered using [Kotlin's inline classes](https://kotlinlang.org/docs/inline-classes.html)? The purpose is really to wrap primitive types to have a language closer to the domain. At the same time it reduces the overhead on the heap for generating new "real" objects.

Reply:
Good point! I will face in technical debt issue!